core/iwasm/fast-jit/jit_compiler.c

```diff
@@ -161,12 +161,12 @@ jit_compiler_compile_all(WASMModule *module)
     if (!(cc = jit_calloc(sizeof(*cc))))
         return false;
 
-    if (!jit_cc_init(cc, 64)) {
-        jit_free(cc);
-        return false;
-    }
-
     for (i = 0; i < module->function_count; i++) {
+        if (!jit_cc_init(cc, 64)) {
```

Comment:
It is better if there is a function like `jit_cc_reset()` to reset `block_stack`, `jit_frame`, every `JitBasicBlock`.

Or else, a combination of `jit_cc_destroy()` and `jit_cc_init()` will do the work.

@wenyongh

Reply:
my bad, `jit_cc_destroy` fits. No necessary to create a `jit_cc_reset()`